src/verify.py

```diff
@@ -0,0 +1,8 @@
+def verify(number_to_search: int) -> bool:
+    static_array = [3, 4, 5, 1, 2, 3, 4, 9, 13, 0]
+
+    should_add_not = '' if number_to_search in static_array else 'not '
+
+    print(f'The number {number_to_search} is {should_add_not}present in the array.')
+    return number_to_search in static_array
```

Comment:
Dato che utilizzi più volte `number_to_search in static_array`, potresti memorizzare il risultato in una variabile locale e poi fare i check con quella. 
Ad esempio 
```py
is_present = number_to_search in static_array
print(f'The number {number_to_search} is {"" if is_present else "not "}present in the array.')
return is_present
```
Se `static_array` fosse estremamente lunga, sarebbe dispendioso fare l'operazione `in` due volte.